batch-1/movie-info/ashish/movie_details.rb

```diff
@@ -0,0 +1,31 @@
+require 'json'
+require 'open-uri'
+puts"Hello,Enter the movie name:\n"
+usr_input=gets.chomp
+usr_input=usr_input.split(" ")# Check whether the entered movie is more than one word
+usr_input=usr_input.join("+")#If yes,add + in between the words.
+movie_url= "http://www.omdbapi.com/?t=#{usr_input}&y=&plot=short&r=json"
+content = open(movie_url).read
+content_hash=JSON.parse(content)
+if content_hash["Response"]=="False"
+	puts 'Please enter a valid movie name.'
+	exit(0)
+end
+html_file=File.new("movie_detail.html","w")
+template_add_data=File.read("template.html")
+template_read=template_add_data.split("\n")
+data=[content_hash["Title"],content_hash["Poster"],content_hash["Genre"],content_hash["Actors"],content_hash["Plot"],content_hash["imdbRating"]]
+repace_list=["{{Title}}","{{Poster}}","{{Genre}}","{{Actors}}","{{Plot}}","{{imdbRating}}"]
+template_read.each_index do |i|
```

Comment:
instead of doing ```.each_index do |i|``` and accessing them later with ```...[i]```, isn't better to do ```template_read.each do |obj|``` and directly access it with ```obj``` variable?

Reply:
baby steps :D